k8s/eks/bash/functions.sh

```diff
@@ -0,0 +1,343 @@
+#!/bin/bash
+# error log prep check
+prep_log_dir(){ 
+mkdir -p ./log/$start-log/
+}
+create_cluster() {
+  eksctl create cluster --name $CLUSTER_NAME --without-nodegroup --region=$REGION
+  echo "cluster_setup_init=true" >> ./.env
+}
+
+remove_aws_node_ds() {
+    kubectl delete daemonset -n kube-system aws-node
+}
+
+add_tigera_operator() {
+    kubectl create -f https://projectcalico.docs.tigera.io/manifests/tigera-operator.yaml
+}
+
+deploy_tigera_operator() {
+    kubectl create -f ./yaml/tigera.yml
+}
+
+deploy_multus_ds() {
+    kubectl apply -f ./yaml/multus-daemonset.yml
+}
+
+clone_multus() {
```

Comment:
Since we are installing multus from local repo, this is not needed anymore.

Reply:
Could you remove it then?

Reply:
Removed from the script.

Reply:
> Also, could you point me at where that installation happens?

Reply:
The fucntion is called `deploy_multus_ds` and in the script it is executed on line 46.

Reply:
Not sure what you mean by that, can you please explain further?

What happens here is that we have a function called `deploy_multus_ds` that essentially deploys the yaml file you linked to the cluster.
And that is how multus is installed/deployed to the cluster.

Reply:
This is resolved now. The copied file - `multus-daemonset.yml` - has been identified and correctly marked as copied form another repo.